x/inflation/keeper/keeper.go

```diff
@@ -28,7 +26,7 @@ type Keeper struct {
 func NewKeeper(
 	storeKey storetypes.StoreKey,
 	cdc codec.BinaryCodec,
-	ps paramtypes.Subspace,
+	authority string,
```

Comment:
Same question as before. Could we use an address as type here (`sdk.AccAddress`)? Or there's a particular reason why we are using `string`
```suggestion
	authority sdk.AccAddress,
```

If this suggestion is committed, we'll need to refactor in other code sections